### ISSUE TITLE
Forum Post Editing: Not Allowed?

### ISSUE THREAD

**Issue by alterniti** [TimMcCool/scratchattach#245]
In the SA wiki you mentioned that making forum posts will never be allowed to prevent spam; however does this also include _editing_ forum posts? 
~~Additionally, is it possible to add the ability to edit your forum signature?~~ (answered)

**Comment by TimMcCool** [comment]
Creating forum posts will never be a scratchattach feature. But editing forum posts doesn't cause spam.
You can update your forum signature using

```py
user = session.connect_user("username") #replace with your username
user.set_forum_signature(text) # Changes the discuess forum signature of the user. New in v1.7.3
```

**Comment by alterniti** [comment]
Ok thanks, I didn’t see the signature editing function.

**Comment by TimMcCool** [comment]
In case you didn't see it, editing forum posts is also already possible, however all forum-related scratchattach features are currently broken - will be fixed in v2.0

**Comment by alterniti** [comment]
Oh ok, yeah I missed that. Thanks :)
